lib/bundix/manifest.rb

```diff
@@ -39,7 +39,7 @@ def to_nix
       sha256 = "<%= gem.source.sha256 %>";
       fetchSubmodules = <%= gem.source.submodules %>;
       <%- elsif gem.source.type == 'gem' -%>
-      remotes = <%= gem.source.remotes.inspect %>;
+      remotes = [<%= [*gem.source.remotes].map(&:dump) * ' ' %>];
```

Comment:
What's the reason for wrapping in an array and splatting here?
Also, might `Array#join` be a bit more readable that `Array#*`?